clang/docs/ReleaseNotes.rst

```diff
@@ -319,6 +319,7 @@ C23 Feature Support
 
 - Clang now supports `N3029 <https://www.open-std.org/jtc1/sc22/wg14/www/docs/n3029.htm>`_ Improved Normal Enumerations.
 - Clang now officially supports `N3030 <https://www.open-std.org/jtc1/sc22/wg14/www/docs/n3030.htm>`_ Enhancements to Enumerations. Clang already supported it as an extension, so there were no changes to compiler behavior.
+- Extend clang's <limits.h> to define *LONG_LONG*_ macros for Android's bionic.
```

Comment:
```suggestion
- Extend Clang's ``<limits.h>`` to define ``LONG_LONG_*`` macros for Android's bionic.
```
Also, this should be moved up under the `C Language Changes` heading (it's currently listed under `C23 Feature Support`).